### ISSUE TITLE
Impossible to open a chat conversation

### ISSUE THREAD

**Issue by willyaranda** [loqui/im#41]
From Contacts, if I tap in a connected contact, there is no opened window for chatting. No error is shown on logcat.

This break Loqui best functionality: chat :)

PS: using latest commit from here and pushed to the phone with the simulator

**Comment by aesedepece** [comment]
Note that it works on app close and reopen.

For some reason, some elements stop receiving tap events after a while. It's the same as #36, which has not been definitively solved.

It has been reported as a issue in Lungo, which seems to be where the problem resides:
https://github.com/tapquo/Lungo.js/issues/211

Frankly, I'm not sure wether it's a Lungo issue or Gaia issue :S

**Comment by kkdeep** [comment]
The issue is still present. Even after closing & reopening i am unable to start a conversation.

**Comment by niktto** [comment]
I have same issue with the marketplace version.

**Comment by marti1125** [comment]
Keon software (Boot2Gecko 1.0.1.0-prerelease)

I test loqui in the emulator, It works perfectly.

$$('section#main article.list ul li').singleTap(function (e){
  Messenger.chatShow(e.currentTarget.dataset.jid);
  alert('chat');
});

I added an alert message to check singleTap Method, in the emulator I saw a message, but in the Keon I didn't see the message.

I think the problem is with http://quojs.tapquo.com/ so 
I created an example using singleTap method it works in both.

I found another bug in Keon, in options Toggle Sound, the screen goes black and It application freezes.
